pyctbgui/services/Acquisition.py

```diff
@@ -656,45 +656,37 @@ def read_zmq(self):
                     # get zoom state
                     viewBox = self.mainWindow.plotAnalogImage.getView()
                     state = viewBox.getState()
-
-                    # get histogram (colorbar) levels and histogram zoom range
-                    levels = self.mainWindow.plotAnalogImage.getHistogramWidget().item.getLevels()
-                    histRange = self.mainWindow.plotAnalogImage.getHistogramWidget().item.getHistogramRange()
-
                     analog_array = np.array(
                         np.frombuffer(data, dtype=np.uint16, count=self.mainWindow.nADCEnabled * self.asamples))
-
                     try:
                         self.mainWindow.analog_frame = decoder.decode(analog_array, self.mainWindow.pixelMapAnalog)
+                        self.plotTab.ignoreHistogramSignal = True
                         self.mainWindow.plotAnalogImage.setImage(self.mainWindow.analog_frame.T)
+
                     except Exception:
                         self.mainWindow.statusbar.setStyleSheet("color:red")
                         message = f'Warning: Invalid size for Analog Image. Expected' \
                                   f' {self.mainWindow.nAnalogRows * self.mainWindow.nAnalogCols} ' \
                                   f'size, got {analog_array.size} instead.'
                         self.updateCurrentFrame('Invalid Image')
+
                         self.mainWindow.statusbar.showMessage(message)
                         print(message)
-                        pass
```

Comment:
was this missed out by mistake?

Reply:
the `pass` statement in python does nothing. it is used usually as a placeholder if we want to have an empty block.
here since the the exception block is not empty we don't need it